lib/jsdom/living/helpers/cors-settings.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+const { asciiLowercase } = require("./strings");
+
+// https://html.spec.whatwg.org/multipage/urls-and-fetching.html#cors-settings-attributes
+const invalidValueDefault = "anonymous";
+const missingValueDefault = "no-cors";
```

Comment:
You could use `null` for this value to mean "No CORS". This way we don't need an additional `no-cors` string. We can also remove `getKeword`, which seems to be unused anyway.